src/common/types/http.d.ts

```diff
@@ -1,7 +1,7 @@
 import HttpMethods from '../constants/HttpMethods';
 import Rest from '../lib/client/rest';
 import ErrorInfo from '../lib/types/errorinfo';
-import http from 'http';
+import http from 'src/common/types/http';
```

Comment:
This should be importing `http` from the node standard library so this change should be reverted